SlevomatCodingStandard/Helpers/ClassHelper.php

```diff
@@ -104,6 +112,39 @@ public static function getTraitUsePointers(File $phpcsFile, int $classPointer):
 		return $useStatements;
 	}
 
+	public static function getPropertiesCount(File $file, int $position): int
+	{
+		return count(self::getProperties($file, $position));
+	}
+
+	/**
+	 * @phpcsSuppress SlevomatCodingStandard.TypeHints.DisallowMixedTypeHint.DisallowedMixedTypeHint
+	 * @param File $file
+	 * @param int $position
+	 * @return mixed[]
+	 */
+	public static function getProperties(File $file, int $position): array
+	{
+		$tokens = $file->getTokens();
+		$token = $tokens[$position];
+		$pointer = $token['scope_opener'];
+
+		self::$propertyList = [];
+
+		while (($pointer = $file->findNext(T_VARIABLE, $pointer + 1, $token['scope_closer'])) !== false) {
+			self::extractPropertyIfFound($file, $pointer);
+		}
+
+		return self::$propertyList;
```

Comment:
I think this code be rewritten as follows:

```php
		return array_filter(
			TokenHelper::findNextAll($file, T_VARIABLE, $pointer + 1, $token['scope_closer']),
			static function ($variablePointer) use ($file) {
				return PropertyHelper::isProperty($file, $variablePointer);
			}
		);
```

This avoids the use of a static variable, and reuses the `TokenHelper::findNextAll` method.